pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -221,6 +220,17 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (result re
 			return reconcile.Result{Requeue: true}, nil
 		}
 		reqLogger.Info("No status cache found")
+	} else {
+		//The status annotation is unneeded, as the status is present, and it will get outdated, so removing it
```

Comment:
So the status annotation will only be sticky when the host is paused? I guess that makes sense.

Reply:
I don't think so, because if we were unable to save the status, then the annotation will still be there the next reconciliation loop and will only be removed once the status has been repopulated properly

Reply:
the idea is to not have an outdated annotation sticking around